csharp-basic-topics/GuidClassInCSharp/GuidClassInCSharpTests/GuidClassInCSharpUnitTests.cs

```diff
@@ -0,0 +1,54 @@
+using static GuidClassInCSharp.GuidClassInCSharpMethods;
+
+namespace GuidClassInCSharpTests
+{
+    public class Tests
+    {
+        [Test]
```

Comment:
Maybe this test would be clearer with a TestCase attribute in place of the Test attribute.
[TestCase("B5724FB4-2FA0-4E68-B108-93C24A1EA6DF")]
public void GivenCheckGuidEqualityMethod_WhenEqualGuids_ThenTrue(string guidValue)